java/test/apps/gui3/dp3/DecoderPro3Test.java

```diff
@@ -13,45 +13,50 @@
  *
  * Description: Tests for the DecoderPro3 application.
  *
- * @author  Paul Bender Copyright (C) 2016
+ * @author Paul Bender Copyright (C) 2016
  */
 public class DecoderPro3Test {
 
     @Test
     public void testCtor() {
-        Assume.assumeFalse(GraphicsEnvironment.isHeadless());        
+        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
         String[] args = {"DecoderProConfig3.xml"};
-        apps.AppsBase a = new DecoderPro3(args){
+        AppsBase a = new DecoderPro3(args) {
             // force the application to not actually start.  
             // Just checking construction.
-            @Override 
-            protected void start(){}
-           @Override
-            protected void configureProfile(){
-                 JUnitUtil.resetInstanceManager();
+            @Override
+            protected void start() {
+            }
+
+            @Override
+            protected void configureProfile() {
+                JUnitUtil.resetInstanceManager();
             }
+
             @Override
-            protected void installConfigurationManager(){
-                 JUnitUtil.initConfigureManager();
-                 JUnitUtil.initDefaultUserMessagePreferences();
+            protected void installConfigurationManager() {
+                JUnitUtil.initConfigureManager();
+                JUnitUtil.initDefaultUserMessagePreferences();
             }
+
             @Override
-            protected void installManagers(){
-                 JUnitUtil.initInternalTurnoutManager();
-                 JUnitUtil.initInternalLightManager();
-                 JUnitUtil.initInternalSensorManager();
-                 JUnitUtil.initRouteManager();
-                 JUnitUtil.initMemoryManager();
-                 JUnitUtil.initDebugThrottleManager();
+            protected void installManagers() {
+                JUnitUtil.initInternalTurnoutManager();
+                JUnitUtil.initInternalLightManager();
+                JUnitUtil.initInternalSensorManager();
+                JUnitUtil.initRouteManager();
+                JUnitUtil.initMemoryManager();
+                JUnitUtil.initDebugThrottleManager();
             }
+
             @Override
-            protected void installShutDownManager(){
-                 JUnitUtil.initShutDownManager();
+            protected void installShutDownManager() {
+                JUnitUtil.initShutDownManager();
             }
         };
         Assert.assertNotNull(a);
         // shutdown the application
-        a.handleQuit();
+        AppsBase.handleQuit();
```

Comment:
This deserves a little more comment. AppsBase.handleQuit() and AppsBase.handleRestart() are both static since they are used by both jmri.App and jmri.AppsBase-derived applications, even though jmri.App-based applications (CATS and PanelPro) do not have an AppsBase instance created while the application is running. They probably should be final methods because of this.